src/CQRS/LeanCode.CQRS.Annotations/PathAliasAttribute.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace LeanCode.CQRS.Annotations;
+
+/// <summary>
+/// Register command/query/operation under additional HTTP path. This could be useful for reorganizing contracts namespaces without introducing a braking change.
```

Comment:
```suggestion
/// Register command/query/operation under additional HTTP path. This could be useful for reorganizing contracts namespaces without introducing a breaking change.
```
I love the driving reference.